programmers/난이도별/level02.[1차]_뉴스_클러스터링/SSSOy.java

```diff
@@ -0,0 +1,61 @@
+import java.util.*;
+
+class Solution {
+    public int solution(String str1, String str2) {
+        int answer = 0;
+        str1 = str1.toLowerCase();
+        str2 = str2.toLowerCase();
+        int str1_len = str1.length();
+        int str2_len = str2.length();
+        
+        List<String> s1 = new ArrayList();
+        List<String> s2 = new ArrayList();
+        
+        for(int i = 0; i < str1_len - 1; i++) {
+            if(str1.charAt(i) >= 'a' && str1.charAt(i) <= 'z' && str1.charAt(i + 1) >= 'a' && str1.charAt(i + 1) <= 'z') {
+                s1.add(str1.substring(i, i + 2));
+            }
+        }
+        for(int i = 0; i < str2_len - 1; i++) {
+            if(str2.charAt(i) >= 'a' && str2.charAt(i) <= 'z'&& str2.charAt(i + 1) >= 'a' && str2.charAt(i + 1) <= 'z')
+                s2.add(str2.substring(i, i + 2));
+        }
+        
+        Collections.sort(s1);
+        Collections.sort(s2);
+        
+        int intersect = 0;
+        int union = 0;
+        
+        Iterator<String> iter1 = s1.iterator();
+        while(iter1.hasNext()) {
+            String s = iter1.next();
+            boolean flag = false;
+            
+            Iterator<String> iter2 = s2.iterator();
+            while(iter2.hasNext()) {
+                if(s.equals(iter2.next())) {
+                    intersect ++;
+                    iter2.remove();
+                    flag = true;
+                    break;
+                }
```

Comment:
```suggestion
                if(!s.equals(iter2.next())) continue;
                intersect ++;
                iter2.remove();
                flag = true;
                break;
```
이런식으로 if로 인하여 indent가 깊어지지 않게 작성해주면 좋습니다!